guestbook/frontend-service.yaml

```diff
@@ -6,6 +6,8 @@ metadata:
     app: guestbook
     tier: frontend
 spec:
+  # comment or delete the following line if you want to use a LoadBalancer
+  type: NodePort 
```

Comment:
Ah, I hadn't considered that. It looks like Minikube will just output <pending> for EXTERNAL-IP if I leave both `type: NodePort` and `type: LoadBalancer` uncommented, but I can otherwise still complete the tutorial. 

Does including both types affect any for the Container Engine side?